index.js

```diff
@@ -0,0 +1,56 @@
+const movesUp = "("
+const movesDown = ")"
+
+
+// First exercise --------------------
+const countSantasMoves = function(inputValue, santasDirection) {
+  const checkSantasMoves = new RegExp(`\\${santasDirection}`, 'g');
+  
+  if (inputValue == null) {
+    return 0
+  }
+
+  let moves = inputValue.match(checkSantasMoves);
+  return moves ? moves.length : 0;
+}
+
+const santasFloor = (inputValue, movesUp, movesDown) => 
```

Comment:
na verdade eu nao fiz com esse if interno aí não... era exatamente isso que queria saber.

E ainda não ta fazendo sentido pra mim sobre como a função vai identificar o que eu to querendo que ela verifique hahaha